db/migrate/20191213094311_add_document_ref_to_whitehall_imports.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddDocumentRefToWhitehallImports < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :whitehall_imports,
+                  :document,
+                  foreign_key: { on_delete: :cascade },
+                  null: true,
+                  index: true
```

Comment:
I don't really think we'll need an index on this as I don''t expect we'll be traversing WhitehallImports via document.